RecoMuon/MuonIsolation/plugins/MuonPFIsolationWithConeVeto.cc

```diff
@@ -59,11 +59,11 @@ bool MuonPFIsolationWithConeVeto::isInIsolationCone(const reco::CandidatePtr& ph
           break;
         }
       }
-      result *= ( is_vertex_allowed );
+      result = is_vertex_allowed;
```

Comment:
I see the point now, thus alternatively one  could read it as `result = result && (is_vertex_allowed)`.

Compiler marks this as suspicious due to `*` operation in boolean context.

Thanks!